style.css

```diff
@@ -389,4 +389,133 @@ span {
   border-radius: 8px;
   padding: 10px 12px;
   margin-right: 15px;
+  float: left;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex: 1fr 1fr 330px;
+}
+
+.contact-heading {
+  display: flex;
+  justify-content: center;
+  margin: 10px;
+  padding: 0;
+}
+
+.contact-h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  gap: 30px;
+  flex-basis: 327px;
+  height: 330px;
+  margin-top: 15px;
+}
+
+.contact-list {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  list-style-type: none;
+  justify-content: center;
+}
+
+.contact-1 {
+  height: 48px;
+  width: 327px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
+}
+
+.contact-2 {
+  height: 48px;
+  width: 327px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
```

Comment:
[ OPTIONAL] 
- [ ] Kindly make sure your page is not overlapping on the right. as you can see
![image](https://user-images.githubusercontent.com/97100091/194086825-88688633-66fc-4cde-9d29-f1a30dd025e0.png)
Make sure you use a relative unit like `%`or `vw`.  As you can notice here your width is to `327px` whereas the minum mobile width is `320px`